charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -426,40 +426,6 @@ data:
         action: drop
 {{- end }}
 
-    # Fetch logs of the tunnel pod (vpn-shoot) via the kube-apiserver, which requires a functional tunnel connection.
-    - job_name: tunnel-probe-apiserver-proxy
-      honor_labels: false
-      metrics_path: /probe
-      params:
-        module:
-        - http_apiserver
-      kubernetes_sd_configs:
-      - role: pod
-        namespaces:
-          names: [ kube-system ]
-        api_server: https://kube-apiserver:443
-        tls_config:
-{{ include "prometheus.tls-config.kube-cert-auth" . | indent 10 }}
-      relabel_configs:
-      - target_label: type
-        replacement: seed
-      - source_labels: [ __meta_kubernetes_pod_name ]
-        action: keep
-        regex: vpn-shoot-(.+)
-      - source_labels: [ __meta_kubernetes_pod_name ]
-        target_label: __param_target
-        regex: (.+)
-        replacement: https://kube-apiserver:443/api/v1/namespaces/kube-system/pods/${1}/log?tailLines=1
-        action: replace
-      - source_labels: [ __param_target ]
-        target_label: instance
-        action: replace
-      - target_label: __address__
-        replacement: 127.0.0.1:9115
```

Comment:
which all parameters should we move to constants in the component file?
like, for deploymentName and serviceName, we have defined them as constants.
But not for some others.

Reply:
Is that because these are the variables that may happen to change in the future, but the rest may not that frequently, so they're hard-coded?